docs/building_workflows.rst

```diff
@@ -0,0 +1,218 @@
+.. include:: links.rst
+
+.. _building_workflows:
+
+###############################
+Custom Reconstruction Workflows
+###############################
+
+QSIRecon workflows are defined in ``json`` files. The :ref:`builtin_workflows`
+can be found in the QSIRecon ``json`` format on
+`github <https://github.com/PennLINC/qsirecon/tree/main/qsirecon/data/pipelines>`_,
+and are a great place to get started with customizing parts of these workflows.
+
+This format has a few key parts.
+
+***********************
+Pipeline-level metadata
+***********************
+
+At the root level of the Recon Spec there are
+
+.. code-block:: json
+
+  {
+    "name": "dsistudio_pipeline",
+    "anatomical": ["mrtrix_5tt_hsvs"],
+    "atlases": ["schaefer100" "schaefer200", "schaefer400", "brainnetome246",
+                "aicha384", "gordon333", "aal116"],
+    "nodes": []
+  }
+
+The ``"name"`` element defines the name of the pipeline. There will be a directory in your working
+directory with this name. The ``"anatomical"`` field lists additional things to compute
+based on the T1w or T2w images in the input data. Currently accepted values are
+
+  * ``"mrtrix_5tt_hsvs"``: A MRtrix_ 5tt segmentation based on [Smith2020]_. This requires :ref:`include_freesurfer`
+
+The ``"atlases"`` field is a list of segmentations (:ref:`connectivity_atlases`) that will be used to create
+:ref:``
+
+
+
+**************
+Pipeline nodes
+**************
+
+
+
+A *node* in the QSIRecon ``"nodes"`` list represents a unit of processing.
+Unlike NiPype nodes, which represent a single call to a commandline program
+or function, a node here represents a set of steps that accomplish a task.
+For example, if you want to perform CSD there are other steps that should
+happen before actually fitting spherical harmonics to the data. The entry
+for this in the ``"nodes"`` list could look like:
+
```

Comment:
One small thing- I like to put each sentence on a separate line, sometimes split on punctuation if it gets too long. I think that leads to cleaner diffs in the long run.